dropwizard-zipkin/src/main/java/smartthings/dw/zipkin/ZipkinConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package smartthings.dw.zipkin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.validation.constraints.NotNull;
+
+public class ZipkinConfiguration {
+
+    private SpanReporterFactory reporter;
+
+    @NotNull
+    private String serviceName;
+
+    private String serviceHost = "127.0.0.1";
+
+    private int servicePort = 8080;
+
+    private float sampleRate = 1.0f;
+
+    private boolean traceId128Bit = false;
+
+    @JsonProperty
+    public SpanReporterFactory getReporter() {
+        return reporter;
+    }
+
+    @JsonProperty
+    public void setReporter(SpanReporterFactory reporter) {
+        this.reporter = reporter;
+    }
+
+    @JsonProperty
+    public String getServiceName() {
+        return serviceName;
+    }
+
+    @JsonProperty
+    public void setServiceName(String serviceName) {
+        this.serviceName = serviceName;
+    }
+
+    @JsonProperty
+    public String getServiceHost() {
+        return serviceHost;
+    }
+
+    @JsonProperty
+    public void setServiceHost(String serviceHost) {
+        this.serviceHost = serviceHost;
+    }
+
+    @JsonProperty
+    public int getServicePort() {
+        return servicePort;
+    }
+
+    @JsonProperty
+    public void setServicePort(int servicePort) {
+        this.servicePort = servicePort;
+    }
+
+    @JsonProperty
+    public float getSampleRate() {
+        return sampleRate;
+    }
+
+    @JsonProperty
+    public void setSampleRate(float sampleRate) {
+        this.sampleRate = sampleRate;
+    }
+
+    @JsonProperty
+    public boolean isTraceId128Bit() {
+        return traceId128Bit;
+    }
+
+    @JsonProperty
```

Comment:
I do not believe you need any of the annotations. Since it is a standard POJO, Jackson will be able to serialize/deserialize all the properties using the name of the field.  Typically, you use those JsonProperty annotations if you want the serialized/deserialized field names to be different from the property names on the POJO.

Feel free to write a test to proof it out.